docs/api/rest_api/rest_api_reference/input/ez-orders.raml

```diff
@@ -76,69 +76,70 @@ displayName: Orders
-                        application/vnd.ibexa.api.OrderGet+json
-                Content-Type:
-                    description: The order schema encoded in XML or JSON format.
-                    example:  |
-                        application/vnd.ibexa.api.OrderGet+json
-            responses:
-                200:
-                    description: OK - loads the order.
-                    body:
-                        application/vnd.ibexa.api.Order+json:
-                            type: Order
-                            example: !include examples/orders/GET/Order.json.example
-                401:
-                    description: Error - the user has no permission to read orders.
-                404:
-                    description: Error - the order does not exist.
-        patch:
-            displayName: Update order
-            description: Updates an order.
-            body:
-                application/vnd.ibexa.api.OrderUpdate+json:
-                    type: OrderUpdate
-                    example: !include examples/orders/PATCH/OrderUpdate.json.example
-            headers:
-                Accept:
-                    description: If set, the updated order is returned in XML or JSON format.
-                    example:  |
-                        application/vnd.ibexa.api.OrderUpdate+json
-                Content-Type:
-                    description: The OrderUpdate schema encoded in XML or JSON format.
-                    example:  |
-                        application/vnd.ibexa.api.OrderUpdate+json
-            responses:
-                200:
-                    body:
-                        application/vnd.ibexa.api.Order+json:
-                            type: OrderUpdate
-                            example: !include examples/orders/GET/Order.json.example
-                400:
-                    description: Error - the input does not match the input schema definition.
-                401:
-                    description: Error - the user is not authorized to update this order.
-                404:
-                    description: Error - the order with the given ID does not exist.
-                406:
-                    description: Error - the order cannot be cancelled due to not-allowed workflow transition
-        post:
-            displayName: Cancel order
-            description: Cancels the given order.
-            responses:
-                200:
-                    description: No content - the given order is cancelled.
-                401:
-                    description: Error - the user is not authorized to delete this order.
-                404:
-                    description: Error - the order with the given ID does not exist.
-                406:
-                    description: Error - the order cannot be cancelled due to not-allowed workflow transition
+    /cancel:
+        /{id}:
+            get:
+                displayName: Load order by ID
```

Comment:
It doesn't make much sense that `/orders/order/cancel/{id}` loads an order.

Reply:
[`/orders/order/{id}`](https://github.com/ibexa/order-management/blob/main/src/bundle/Resources/config/routing_rest.yaml#L9) should be kept.

[`/orders/order/cancel/{id}`](https://github.com/ibexa/order-management/blob/main/src/bundle/Resources/config/routing_rest.yaml#L39) is a whole new route.

Reply:
It should looks like
```yaml
displayName: Orders

/orders:
    post:
        …
    get:
        …
/order:
    /{identifier}:
        get:
            …

    /{id}:
        get:
            …
        patch:
            …
        post:
            …
    /cancel/{id}:
        post:
            displayName: Cancel order
            …
```

Reply:
In fact, it's just the insertion of `    /cancel/{id}:` at line 133